lib/tlsca/ca.go

```diff
@@ -287,6 +292,11 @@ var (
 	// ActiveRequestsASN1ExtensionOID is an extension OID used when encoding/decoding
 	// active access requests into certificates.
 	ActiveRequestsASN1ExtensionOID = asn1.ObjectIdentifier{1, 3, 9999, 2, 8}
+
+	// DisallowReissueASN1ExtensionOID is an extension OID used to flag that a
+	// requests to generate new certificates using this certificate should be
+	// denied.
+	DisallowReissueASN1ExtensionOID = asn1.ObjectIdentifier{1, 3, 999, 2, 9}
```

Comment:
Are you missing a `9` here?
```suggestion
	DisallowReissueASN1ExtensionOID = asn1.ObjectIdentifier{1, 3, 9999, 2, 9}
```

Reply:
Oh geez, yup, thanks!